qemu/tests/timedrift_check_when_hotplug_vcpu.py

```diff
@@ -41,6 +42,7 @@ def run(test, params, env):
     query_times = int(params.get("query_times", "4"))
     query_internal = float(params.get("query_internal", "600"))
     drift_threshold = float(params.get("drift_threshold", "3"))
+    hotplug_cmd = params.get("cpu_hotplug_cmd", "")
```

Comment:
Should we use  params.get("cpu_hotplug_cmd") ?

Reply:
Updated